app/components/images/imagesController.js

```diff
@@ -59,7 +59,28 @@ function ($scope, $state, Config, Image, ImageHelper, Messages, Pagination) {
     });
   };
 
-  $scope.removeAction = function () {
+  $scope.confirmRemovalAction = function (force) {
+    ModalService.confirm({
```

Comment:
Hi,

I understand your point, but passing such parameters as arguments will make this module hard to evolve.

I would prefer passing parameters as an object: 

```
ModalService.confirmModal({
   title: 'Are you sure?',
   message: 'Forcing the removal of the image will remove the image even if it has multiple tags or if it is used by stopped containers.',
  confirm: {
    label: 'Remove the image'
  },
  cancel: {
    label: 'cancel'
  }
})
```
To make my point clear, in our situation, the className of the confirm button is red for danger.
This may not be the case, if later, for example, we want to use this service to confirm a password change.

Passing an object will make it simpler to extend with additional parameters.